src/openloco/game_commands.h

```diff
@@ -26,6 +26,18 @@ namespace openloco::game_commands
         do_command(21, regs);
     }
 
+    inline void do_46(uint8_t bl, uint16_t cx, uint16_t ax, uint32_t edx, uint32_t ebp, uint32_t edi)
```

Comment:
Would rather see this as a single game command with memcpys instead of pointer magic.

e.g. `do_46(town_id cx, char[36])`

Reply:
I would prefer to see that, too, but preferably not in this PR.

Reply:
I would prefer to see that, too, but preferably not in this PR. This will be difficult one to tackle, as these commands are (ultimately) networked. The same char pointer will not be valid for other players.